src/main/java/nl/tudelft/ti2206/group9/level/State.java

```diff
@@ -65,7 +66,7 @@ public static void resetAll() {
         setCoins(0);
         highscore = 0;
         previousDistance = 0;
-        skin = Skin.getNoob();
+        skin = Skin.loadSkinsToList().get(0);
```

Comment:
Is this a smart thing to do? This way you hardcode the position of the Noobskin in the List.

<review